src/plugins/ResizeReorder/ReorderHandle.js

```diff
@@ -0,0 +1,293 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReorderHandle
+ * @typechecks
+ */
+
+import React from 'react';
+import cx from '../../vendor_upstream/stubs/cx';
+import PropTypes from 'prop-types';
+import DOMMouseMoveTracker from '../../vendor_upstream/dom/DOMMouseMoveTracker';
+import { PluginContext } from '../../Context';
+
+const DRAG_SCROLL_SPEED = 15;
+const DRAG_SCROLL_BUFFER = 100;
+
+class ReorderHandle extends React.PureComponent {
+  /**
+   * Instance of DOMMouseMoveTracker to capture mouse events
+   * @type {DOMMouseMoveTracker}
+   */
+  mouseMoveTracker = null;
+
+  /**
+   * Displacement of reorder cell while reordering
+   * @type {number}
+   */
+  cursorDeltaX = 0;
+
+  /**
+   * Set true while reordering
+   * @type {boolean}
+   */
+  isReordering = false;
+
+  /**
+   * Frame Id of requested animation frame
+   * @type {number | null}
+   */
+  frameId = null;
+
+  /**
+   * Initial position of scroll
+   * @type {number}
+   */
+  scrollStart = 0;
+
+  /**
+   * Left of column
+   * @type {number}
+   */
+  originalLeft = 0;
+
+  render() {
+    const style = {
+      height: this.props.height,
+    };
+
+    return (
+      <div
+        className={cx({
+          'fixedDataTableCellLayout/columnReorderContainer': true,
+          'fixedDataTableCellLayout/columnReorderContainer/active': false,
+        })}
+        onMouseDown={this.onMouseDown}
+        onTouchStart={this.onTouchStart}
+        onTouchEnd={this.onTouchEnd}
+        onTouchMove={this.onTouchMove}
+        style={style}
+      ></div>
+    );
+  }
+
+  onTouchStart = (ev) => {
+    if (!this.props.touchEnabled) {
+      return;
+    }
+    this.onMouseDown(ev);
+  };
+
+  onTouchEnd = (ev) => {
+    if (this.props.touchEnabled) ev.stopPropagation();
+  };
+
+  onTouchMove = (ev) => {
+    if (this.props.touchEnabled) ev.stopPropagation();
+  };
+
+  /**
+   *
+   * @param {MouseEvent} event
+   */
+  onMouseDown = (event) => {
+    this.props.toggleCellsRecycling(false, this.props.columnKey);
```

Comment:
This is required because of this [recyclable cells issue](https://docs.google.com/document/d/1GawNXJvLniu2DlzoW8HXl51_O5tQ0MEp07wTsswajTQ/edit?disco=AAAAL2Wz8b4). This is added to have backward compatibility. For new versions, users have to manually toggle allowCellsRecycling as shown in